src/main/java/org/fundacionjala/coding/rodrigo/movierefactor/Rental.java

```diff
@@ -0,0 +1,84 @@
+package org.fundacionjala.coding.rodrigo.movierefactor;
+
+/**
+ * Rental for the movies.
+ */
+public final class Rental {
+    private Customer customer;
+    private String bill;
+    private double totalAmount;
+    private int totalFRP;
+
+    /**
+     * Constructor.
+     *
+     * @param customer Input customer.
+     */
+    Rental(final Customer customer) {
+        this.customer = customer;
+        totalAmount = 0;
+        totalFRP = 0;
```

Comment:
These assignations are redundant.
Integer attributes of class are 0 by default.